.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,10 @@
-# Pull request template
+# $PULL_REQUEST_TITLE
 
-<PULL REQUEST TITLE>
+**Implemented:**
+- your_feature_1
 
-Implemented:
-- <Features implemented>
+**Fixed:**
+- your_bugfix_1
 
-Fixed:
-- <Fixes>
-
-<Additional notes>
+**Additional notes:**
+Wolvenkit is awesome :)
```

Comment:
This is cute, but can we actually change it to maybe "closes issue_no" or something? because I tend to forget that I need to include it here...

Reply:
Sure thing :D